.changeset/giant-cobras-tease.md

```diff
@@ -0,0 +1,6 @@
+---
+"@rhds/elements": patch
+---
+
+`<rh-subnav>`:
+  - Added `rh-subnav-lightdom.css` to reduce layout shift before element is defined
```

Comment:
```suggestion
`<rh-subnav>`: Added `rh-subnav-lightdom.css` to reduce layout shift before element is defined
```

individual changesets already render as markdown list items in the changelog, so no need to break them out.

if your changelog does have a list, then the first like would be the top-level list item, and subsequent list items would be 2nd level, etc.